### ISSUE TITLE
Limear Regression diverges

### ISSUE THREAD

**Issue by SG-391** [rslim087a/PyTorch-for-Deep-Learning-and-Computer-Vision-Course-All-Codes-#1]
Hi, I am using exactly the code as per "Linear_Regression.ipynb" but it diverges! Any idea? Is the code still valid?

Thanks,
Serge

**Comment by neeveermoree** [comment]
I have solved this problem by dramatically decreasing the learning rate. In my code lr = 0.00001 works perfectly.

**Comment by SG-391** [comment]
That works, thanks
